pkg/install/app/logservice.go

```diff
@@ -102,6 +88,15 @@ func ConfigureLogServiceContainer(coreobj *obj.Core, c *corev1.Container) {
 		env = append(env, core.Spec.LogService.Env...)
 	}
 
+	// if core.Spec.Vault.LogServicePath != "" {
```

Comment:
Is this needed?

Reply:
no, I'm need to remove this still. good catch!